dividend_collection.py

```diff
@@ -0,0 +1,29 @@
+from dividend_payment import DividendTransaction
+
+
+class DividendCollection:
+    def __init__(self):
+        self.payments = []
+
+    def append(self, dividend_payment: DividendTransaction):
+        self.payments.append(dividend_payment)
+
+    def is_empty(self):
+        return not self.payments
+
+    def get_security(self):#get_fund_names
+        fund_names = []
+        for payment in self.payments:
+            if not payment.fund_name in fund_names:
+                fund_names.append(payment.fund_name)
+        return fund_names
```

Comment:
Here the function is called `get_security()` but in the `Dividend` class we call it `fund_name`. Let's use the same name.

I would say let's call the function `get_fund_names()` so it has the same name we are using in the `TransactionCollection`.